CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🐞 Continuously importing JSON, CSV, and Syslog events now properly works at
+  low rates (e.g., when batched over the network).
```

Comment:
Can we have a clear description of the bug first? It's not apparent to me what was the issue, i.e., was the opposite the bug or something related?

Reply:
This ports the fix from #750 over to other readers. The timeout ensures that we continue when some events were already handled, but the input stream stalls.

Reply:
Actually at "low rates" this is actually less severe, since at least we will eventually get all data. In this case we could just reduce the table slice size to match the input rate. 

The worse bug is when the input is already finished, but we didn't get EOF (e.g. because its going through some pipe indirections), then the data that is already parsed in the partial table slice is just swallowed and is never forwarded to index/archive.

Reply:
I have improved the changelog wording accordingly.